dudley-codes/habitLoop
# Details Expand/Collapse

# Description
Added a feature that expands and collapses the details for a habit.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Click arrow next to a habit, it should expand details. Click it again and it should collapse details.
# Checklist:
